x-pack/plugins/cloud_security_posture/public/pages/configurations/utils/create_detection_rule_from_finding.ts

```diff
@@ -50,9 +50,23 @@ const CSP_RULE_TAG_DATA_SOURCE_PREFIX = 'Data Source: ';
 
 const STATIC_RULE_TAGS = [CSP_RULE_TAG, CSP_RULE_TAG_USE_CASE];
 
+/*
+ * Returns an array of CspFinding tags that can be used to search and filter a detection rule
+ */
+export const getFindingsDetectionRuleSearchTags = ({ rule }: CspFinding) => {
```

Comment:
nit: we refer here to Misconfigurations as Findings, but in general Findings are both misconfigs and vulnerabiltiies. Probably out of the scope for this bugfix, but would be good to think about clear distinction between Misconfigurations and Vulnerabilities in the code and refer to findings only when smth applies to both